tests/sles4sap/sap_suse_cluster_connector.pm

```diff
@@ -65,15 +65,17 @@ sub run {
     }
 
     # List nodes
-    my @resources = get_var('NW') ? ('ip', 'fs', 'sap') : ('ip', 'SAPHanaTopology', 'SAPHana');
+    my @hana_resources = get_var('USE_SAP_HANA_SR_ANGI') ? ('ip', 'SAPHanaFil', 'SAPHanaTpg', 'SAPHanaCtl') : ('ip', 'SAPHanaTpg', 'SAPHana');
```

Comment:
```suggestion
    my @hana_resources = ('ip', 'SAPHanaTpg');
    if (get_var('USE_SAP_HANA_SR_ANGI')) {
        push @hana_resources, 'SAPHanaFil', 'SAPHanaCtl';
     } else {
        push @hana_resources, 'SAPHana';
     }
```

Reply:
No, no, NO! No way! Go to @lpalovsky and solve this with him.

Reply:
ok